examples/pytorch/distributed_stl10/pytorch_distributed_stl10_example.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+#SBATCH --job-name=pytorch_stl10     # job name
+#SBATCH --ntasks=2                   # number of MP tasks
```

Comment:
What do you think about also adding an example that use one or two entire nodes ?

Reply:
The README should now better reflect how to do that.